README.md

```diff
@@ -76,6 +62,34 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Use Base Profiles to access API Mediation Layer
+
+Leverage existing base profiles with a token to log in to API Mediation Layer.
```

Comment:
Leverage existing base profiles with a token to access z/OSMF via the API Mediation Layer.